src/flows/shared/steps/OrganizeCollection/Sidebar/Sidebar.tsx

```diff
@@ -125,22 +153,30 @@ function Sidebar({ tokensRef, sidebarTokens, queryRef }: Props) {
         {!isSearching && (
           <>
             <SidebarChainSelector
+              isTezosAccountConnected={isTezosAccountConnected}
               queryRef={query}
               selected={selectedChain}
               onChange={setSelectedChain}
             />
-            <AddBlankSpaceButton onClick={handleAddBlankBlockClick} variant="secondary">
-              ADD BLANK SPACE
-            </AddBlankSpaceButton>
+            {isTezosAccountConnected && (
+              <AddBlankSpaceButton onClick={handleAddBlankBlockClick} variant="secondary">
+                ADD BLANK SPACE
+              </AddBlankSpaceButton>
+            )}
           </>
         )}
       </StyledSidebarContainer>
-      <SidebarTokens
-        isSearching={isSearching}
-        tokenRefs={nonNullTokens}
-        selectedChain={selectedChain}
-        editModeTokens={tokensToDisplay}
-      />
+
+      {isTezosAccountConnected ? (
+        <SidebarTokens
+          isSearching={isSearching}
+          tokenRefs={nonNullTokens}
+          selectedChain={selectedChain}
+          editModeTokens={tokensToDisplay}
+        />
+      ) : (
+        <AddWalletSidebar queryRef={query} />
```

Comment:
given the change above, we probably want to make this component to be more flexible to say "you do not own any {chain} NFTs".

we'll also want the CTA to say "connect {chain} wallet", but if chain is POAP it should say ethereum (or "ETH" if the button is too long)